src/main/webapp/index.html

```diff
@@ -7,14 +7,15 @@
     <link rel="stylesheet" href="/styles/fonts.css">
     <link rel="stylesheet" href="/styles/common.css">
     <link rel="stylesheet" href="/styles/login.css">
+    <script src="/scripts/login.js"></script>
   </head>
-  <body>
+  <body onload=checkLogin()>
```

Comment:
Don't we need this check in all the pages? If we don't have this check in some pages, the user will be able to access those pages without login, right?

If we do need this check in all the pages, I would recommend to link/import the `login.js` in all pages, have `document.onload` event listener in the `js` file, and have a dedicated `login.html` to redirect the user to when they are not logged in.

Reply:
I changed it to an onload function as suggested, but sadly there is no way for me to test this. Since we need to be logged in to access the pages.

Reply:
Yes! forgot to remove it